### ISSUE TITLE
Developer Keys Deprecated

### ISSUE THREAD

**Issue by scburns123** [aevernon/commission_junction#9]
Unable to use this gem as the Developer Keys are no longer issued (source: https://developers.cj.com/authentication/overview).  Possible to update for the new Personal Access Tokens?

**Comment by brandonbyskov** [comment]
Just to provide some information about recent changes that are required when using personal access tokens:

- the word `Bearer` must be added into the `Authorization` header.
- some of the APIs now have a required `requestor-cid` (company ID) parameter.

Existing integrations using the developer keys should continue to work the same way the did in the past.

**Comment by aevernon** [comment]
Thank you for this information. Would you be willing to send a patch?

**Comment by brandonbyskov** [comment]
No, I'm not a ruby developer. I'm just an engineer from CJ trying to help out.
